docs/content/Getting-Started/Core/02-Create-a-project.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Create a project
+permalink: /getting-started/core/create-a-project
+category: Getting Started
+subCategory: Core
+menuOrder: 4
+isDisableFeedbackBlock: true
+---
+
+## Scaffold a project
+
+To create a new project, run the following command from your terminal:
+
+```bash
+npx cubejs-cli create my-first-cube-project
+```
+
+This command will create a new directory with the project name and install any
+necessary dependencies.
+
+## Start the development server
+
+From the newly-created project directory, run the following command to start
+Cube:
+
+```bash
+docker-compose up -d
+```
+
+## Connect a data source
+
+<InfoBox>
+
+Want to use a sample database instead? Use the credentials below:
+
+<br />
+
+| Field    | Value              |
```

Comment:
I think credentials might be move a few paragraphs down where they are actually needed.

Reply:
Also, let's specify it's Postgres.